### PR TITLE
fix(ProductHero): text vertical spacing

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* ProductHero: fix text stack spacing

image

image

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4354** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4354?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/ProductPage/PurchaseForm/ProductHero/ProductHero.css.ts [hunk 1]
```diff
@@ -13,6 +13,9 @@ export const pillow = style({
   marginInline: 'auto',
 })
 
-export const textWrapper = style({
-  rowGap: theme.space.sm,
-})
+export const textWrapper = style([
```
Comment (Youakeem) [inline apps/store/src/components/ProductPage/PurchaseForm/ProductHero/ProductHero.css.ts:16]: Feels like a great candidate for the `yStack` pattern

Comment (guilhermespopolin) [reply apps/store/src/components/ProductPage/PurchaseForm/ProductHero/ProductHero.css.ts:16]: Yes, let's do it.

Comment (guilhermespopolin) [reply apps/store/src/components/ProductPage/PurchaseForm/ProductHero/ProductHero.css.ts]: Update it.

**Review by alebedev — APPROVED** [review]

**Review by Youakeem — APPROVED** [review]

**Pull request merged**